src/test/java/API/BackendTest.java

```diff
@@ -0,0 +1,96 @@
+package API;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+//Basic API test, currently the tests are flaky due to the use
+//of hardcoded ID's, to be changed when the UI tests using User class
+//are fully developed and implemented
+
+public class BackendTest {
+@BeforeClass
+public static void setup() {
+RestAssured.baseURI = "http://localhost:8080";
+}
+
+@Test
+public void testGetAllUsers() {
+given()
```

Comment:
Can you create a service class that will contain all the interactions with the API? Also we will need Validator class to validate the responses.

Reply:
Also it will be good to use Cucumber instead of pure JUnit